pkg/provision/providers/qemu/launch.go

```diff
@@ -461,3 +462,32 @@ func waitForFileToExist(path string, timeout time.Duration) error {
 		time.Sleep(100 * time.Millisecond)
 	}
 }
+
+func checkPartitions(config *LaunchConfig) (diskBootable bool, err error) {
```

Comment:
probably rename to `checkPartition` and pass in a string since this is checking only a string

Reply:
Renamed 👍  The function is utilizing both `config.DiskPaths` and `config.DiskBlockSizes` so I think it makes sense to keep the config as a param